.env.sample

```diff
@@ -2,13 +2,10 @@
 VUE_APP_VAULT_CONTEXT_NAME=Verida: Vault
 
 # Verida app context name. (Required)
-VUE_APP_VERIDA_CONTEXT_NAME=Verida:Account Explorer
+VUE_APP_VERIDA_CONTEXT_NAME=Verida: Network 
```

Comment:
Why "Network"?
The app is an explorer, not a network itself.
I'm ok to remove the "Account" from the previous name, to make it more generic.
So "Verida: Explorer" or potentially "Verida: Network Explorer", but "Verida: Network" is weird.

Reply:
following the name of the demo which was changed from verida account explorer to verida Network

will change the context name to   "Verida: Network Explorer" this makes more sense.

Reply:
updated